components/brave_ads/core/internal/common/time/time_util.cc

```diff
@@ -14,7 +14,7 @@ namespace brave_ads {
 
 namespace {
 
-bool g_from_local_exploded_failed = false;
+bool g_from_local_exploded_failed_for_testing = false;
```

Comment:
Maybe a follow up, but do we still need this? Did Chromium fix the upstream issue?